ckanext/ckanext-sixodp_ui/ckanext/sixodp_ui/templates/page.html

```diff
@@ -86,18 +86,7 @@ <h1>My page content</h1>
                   #}
                   {% block primary_content %}
                     <article class="module">
-                      {% block page_header %}
-                        <header class="module-content page-header">
-                          {% if self.content_action() | trim %}
-                            <div class="content_action">
-                              {% block content_action %}{% endblock %}
-                            </div>
-                          {% endif %}
-                          <ul class="nav nav-tabs">
-                            {% block content_primary_nav %}{% endblock %}
-                          </ul>
-                        </header>
-                      {% endblock %}
+                      {% block page_header %}{% endblock %}
```

Comment:
Moved page_header block and its contents in read_base.html.

Reply:
This probably brake other pages that assume that page.html still has the page_header content